src/main/webapp/style.css

```diff
@@ -0,0 +1,68 @@
+#content {
+  margin-left: auto;
+  margin-right: auto;
+  width: 900px;
+}
+
```

Comment:
Optional: I'd suggest removing the extra newline in between css declarations for sake of condensing. Whatever you choose to, we'll all need to follow suit.

Reply:
That's weird on my end there isn't an extra newline but I'm pushing it again rn so maybe it will update that.

Reply:
css-selector {

}
*this line*
another-css-selection { ...